src/main/java/core/basesyntax/MainApp.java

```diff
@@ -2,4 +2,13 @@
 
 public class MainApp {
 
+    public static void mail (String[] args) {
```

Comment:
The method name `mail` seems to be a typo. It should be `main` to serve as the entry point of the application.